homeassistant/components/influxdb/sensor.py

```diff
@@ -204,6 +226,15 @@ def __init__(self, hass, influx, query):
                 query.get(CONF_GROUP_FUNCTION),
             )
 
+        if query[CONF_LANGUAGE] == LANGUAGE_INFLUXQL_RAW:
```

Comment:
```suggestion
        elif query[CONF_LANGUAGE] == LANGUAGE_INFLUXQL_RAW:
```
This code will break all InfluxV2 users right now since you've made this a separate if block with an else. For InfluxV2 users it will fall to the else and try and make an `InfluxQLSensorData` for each item in `queries_flux` which won't work.

Reply:
Right, basic mistake, thanks ...